common/changes/office-ui-fabric-react/syncScroll_2018-11-27-13-32.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "sync scroll after sticky forceUpdate",
```

Comment:
Can you elaborate the comment in a more descriptive way what is the change about? This goes to release notes that are read by all consumers and your comment doesn't say them much what they get with a new release. A good way to start would be 'Sticky: do this to fix that...`

Reply:
1) If the distanceFromTop is changed, a forceUpdate would happen.
2) The component would be re-rendered but its scrollLeft would not be synced to scrollablepane content container.
[Here is an example.](https://microsoft-my.sharepoint.com/:f:/p/hatrived/EvtojtX22G1EjEhIrZz_sREBLXCZXNK4JKFIqvgqrj0NOQ?e=SiV5H6)

Reply:
@tharshada thanks for the video demo. It clearly shows there is a bug. Could you please take this 2 points you stated above, combine them and make it as the comment for your changelog?
@leddie24 or @ThomasMichon  can you please review if this bug would be fixed by this PR and if it's not introducing some performance regression. Thanks.

Reply:
Thanks a lot for the change.